<launch>
        <!-- Make sure to put the handle into the namespace of the controller we want to use -->
        <group ns="my_cartesian_motion_controller" >

                <!-- Load an instance of a motion handle -->
                <node name="motion_control_handle"
                pkg="cartesian_controller_handles"
                type="motion_control_handle_node"
                respawn="false"
                output="screen" >
                <param name="target_frame_topic" value="target_frame"/>
                </node>

        </group>

</launch>
